docs/self-managed/identity/orchestration-identity/installation.md

```diff
@@ -16,33 +19,74 @@ Following the default installation will result in a cluster with the following:
-
-### Configure basic authentication
+5. An `admin` role with read, create, update, and delete permissions for other roles, which is applied to the initial `demo` user
 
 To make changes to the initial configuration, add the desired values via your `application.yaml` or environment variables according to the available [configuration properties](./configuration.md).
 
-For example, to enable basic authentication and create an initial user, the following is required:
+### Configure an initial user
+
+The initial user created by the application will be assigned the `admin` role, and can be used for authentication to the web applications and additional role management.
+
+To create a unique initial user, the following is required in your `application.yaml` or environment variables:
+
+<Tabs>
+  <TabItem value="helm" label="Helm properties" default>
 
 ```yaml
-camunda.security:
-  authentication.unprotected-api: false
-  initialization.users[0].username: <Your chosen username>
-  initialization.users[0].password: <Your chosen password>
-  initialization.users[0].name: <The name of the first user>
-  initialization.users[0].email: <The email address of the first user>
+camunda:
+  security:
+    initialization:
+      users:
+        - username: <Your chosen username>
+          password: <Your chosen password>
+          name: <The name of the first user>
+          email: <The email address of the first user>
+```
+
+  </TabItem>
+<TabItem value="env" label="Environment variables" default>
+
+```shell
+CAMUNDA_SECURITY_INITIALIZATION_USERS[0]_USERNAME=<Your chosen username>
+CAMUNDA_SECURITY_INITIALIZATION_USERS[0]_PASSWORD=<Your chosen password>
+CAMUNDA_SECURITY_INITIALIZATION_USERS[0]_NAME=<The name of the first user>
+CAMUNDA_SECURITY_INITIALIZATION_USERS[0]_EMAIL=<The email address of the first user>
 ```
 
-### Enable authorizations
+  </TabItem>
+</Tabs>
 
-To work with authorizations, API authentication and authorization enforcement must be enabled. The following minimal `application.yaml` shows the required configuration for the APIs and authorizations:
+### Enable API authentication and authorizations
+
+By default, basic authentication is enabled on the Camunda web applications, but the API is unprotected, and authorizations are disabled. API protection and authorizations can both be enabled by modifying your `application.yaml` or environment variables:
+
+<Tabs>
+  <TabItem value="helm" label="Helm properties" default>
 
 ```yaml
-camunda.security:
-  authentication.unprotected-api: false
-  authorizations.enabled: true
+camunda:
+  security:
+    authentication:
+      unprotected-api: false
+    authorizations:
+      enabled: true
+```
+
+  </TabItem>
+<TabItem value="env" label="Environment variables" default>
+
+```shell
+CAMUNDA_SECURITY_AUTHENTICATION_UNPROTECTED-API=false
+CAMUNDA_SECURITY_AUTHORIZATIONS_ENABLED=true
 ```
 
+  </TabItem>
+</Tabs>
+
+:::note
+To enable authorizations, API protection must also be enabled.
```

Comment:
👍🏻 this is a great point to note down, in the future we will look to make the application more defensive during start up but its super important to note this otherwise you can get into a broken state :)